cinn/ir/ir_schedule.cc

```diff
@@ -1509,6 +1510,19 @@ void ScheduleImpl::Annotate(const Expr& block, const std::string& key, const att
   this->Replace(block, copied_block);
 }
 
+void ScheduleImpl::Unannotate(const Expr& source_node, const std::string& ann_key) {
+  CHECK(source_node.As<ir::ScheduleBlockRealize>());
+  CHECK(source_node.As<ir::ScheduleBlockRealize>()->schedule_block.As<ir::ScheduleBlock>());
+  auto block           = const_cast<Expr&>(source_node);
+  auto* schedule_block = block.As<ir::ScheduleBlockRealize>()->schedule_block.As<ir::ScheduleBlock>();
```

Comment:
```suggestion
void ScheduleImpl::Unannotate(Expr& block, const std::string& ann_key) {
  CHECK(source_node.As<ir::ScheduleBlockRealize>());
  CHECK(source_node.As<ir::ScheduleBlockRealize>()->schedule_block.As<ir::ScheduleBlock>());
  auto* schedule_block = block.As<ir::ScheduleBlockRealize>()->schedule_block.As<ir::ScheduleBlock>();
```
我们这边需求文档接口写得不够准确，辛苦更新：
1. 当前应该只支持修改ScheduleBlockRealize，因此参数名可为block
2. block可被修改的话输入参数引用不加const

Reply:
done